cartoframes/data/observatory/catalog/subscription_info.py

```diff
@@ -3,6 +3,12 @@
 
 
 class SubscriptionInfo(object):
```

Comment:
This class is not exposed in the reference, should we?

Reply:
I'm going to expose it 👍